samples/CustomMiddleware/HttpFunction.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace CustomMiddleware
+{
+    public class HttpFunction
+    {
+        [Function("HttpFunction")]
+        public static HttpResponseData Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            FunctionContext context)
+        {
+            var logger = context.GetLogger<HttpFunction>();
+
+            // Get the item set by the middleware
+            if (context.Items.TryGetValue("middlewareitem", out object value) && value is string message)
```

Comment:
Could you do `context.Items.TryGetValue("middlewareitem", out string value)` right away?

Reply:
Absolutely. This is what you end up with when you decide last minute on a Friday; “let me write this sample really quick” and end up refactoring some code as you’re cleaning up for PR😆

Reply:
Actually, I take that back 😄 

I remember this actually being the syntax I though it would be nice, but we're working with an `IDictionary<object, object>` here, that that won't work. When I read the comment I immediately thought I had spaced and this was working on string values (same mistake as originally described, just on the opposite end, where I decided to respond to GH reviews too early in the morning :) )

The sample shows how you'd ensure you're working with a valid string that is not `null` in a simple statement here.